packages/create-cloudflare/src/frameworks/gatsby/index.ts

```diff
@@ -37,6 +37,9 @@ const generate = async (ctx: PagesGeneratorContext) => {
 const config: FrameworkConfig = {
 	generate,
 	displayName: "Gatsby",
+	configure: async (ctx: PagesGeneratorContext) => {
```

Comment:
If this doesn't make a difference let's maybe remove, since it is somewhat expensive

Reply:
yeah good point, sorry I forgot to take it back, thanks 🙂

Reply:
removed 🙂👍